server/index.controller.js

```diff
@@ -22,11 +22,22 @@ function getArticleById (req, res, next) {
     if (article.rows.length === 0) {
       next({status: 404, msg: "not found"})
     }
-    res.status(200).send({article: article.rows})
+    res.status(200).send({article: article.rows[0]})
   })
   .catch((err) => {
     next(err)
   })
 }
 
-module.exports = {getTopics, getEndpoints, getArticleById}
\ No newline at end of file
+function getArticles (req, res, next) {
+  selectArticles()
+  .then((articles) => {
+    const noBody = articles.rows.map((article) => {
+      delete article.body
+      return article
+    })
```

Comment:
This does work, but could and should instead be handled in the model by your sql query instead.